jekyll/_cci2/build-processing.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: classic-docs
+title: "Enabling Build Processing"
+short-title: "Enabling Build Processing"
+description: "How to enable the preview of build processing"
+categories: [settings]
+order: 1
+---
+
+# CircleCI Build Processing
+
+This document describes how to enable the preview build processing engine if you need to trigger workflows from the CircleCI API.
+
+## Getting Started
+Enable build processing at the bottom of the Advanced section of Settings page for your project in the CircleCI app. If you don't see this setting, it should appear in the next few days for you as we roll it out to more and more organizations.
+
+The new build processing feature enables use of the new [API endpoint to trigger builds with workflows]({{ site.baseurl }}/api/v1-reference/#projects) and the following use cases:
+
+- GitHub and BitBucket support including API and webhooks
```

Comment:
We can probably remove this from here - it was in the preview docs because it wasn't yet done, but as a new doc it's not really a "feature" -- is exclusion of the 2.1 config features purposeful because they are still in preview?

Reply:
yep, let's document the 2.1 config features after we get this info out about workflows API trigger

Reply:
removed first bullet